### PR TITLE
extend stringToString pflag binding to stringToInt pflag

### PR DESCRIPTION
**Pull request by vorishirne**
Support for pflag's StringToString was added long back, but Viper Still couldn't bind to pflag StringToInt. Addid that support.

### CODE REVIEW

**Comment by vorishirne** [comment]
@sagikazarmark can u look into this

File: viper.go [hunk 1]
```diff
@@ -1373,7 +1373,7 @@ func readAsCSV(val string) ([]string, error) {
 
 // mostly copied from pflag's implementation of this operation here https://github.com/spf13/pflag/blob/master/string_to_string.go#L79
 // alterations are: errors are swallowed, map[string]interface{} is returned in order to enable cast.ToStringMap
-func stringToStringConv(val string) interface{} {
+func csvKeyValueConv(val string) interface{} {
```
Comment (sagikazarmark) [inline viper.go:1376]: Wouldn't `stringToInterfaceCong` be a better name?

**Review by sagikazarmark — CHANGES_REQUESTED** [review]
Thanks for sending a PR! Can you please take a look at the comment I left?

File: viper.go (hunk 1 above)
Comment (vorishirne) [reply viper.go:1376]: Sure let me do that

**Comment by vorishirne** [comment]
@sagikazarmark I mistakenly closed this MR. I have created a new MR

https://github.com/spf13/viper/pull/1462

Your suggestion has been implemented there.

File: viper.go (hunk 1 above)
Comment (vorishirne) [reply viper.go]: Done. But moved to another MR. Some issue happened in this MR
https://github.com/spf13/viper/pull/1462

**Pull request closed without merging**